libs/src/api/account.js

```diff
@@ -111,21 +111,21 @@ class Account extends Base {
       CREATE_USER_RECORD: 'CREATE_USER_RECORD',
       HEDGEHOG_SIGNUP: 'HEDGEHOG_SIGNUP',
       UPLOAD_PROFILE_IMAGES: 'UPLOAD_PROFILE_IMAGES',
-      ADD_USER: 'ADD_USER',
-      UPLOAD_METADATA_AND_UPDATE_ON_CHAIN: 'UPLOAD_METADATA_AND_UPDATE_ON_CHAIN'
+      ADD_USER: 'ADD_USER'
     }
     let phase = ''
     let userId
 
     try {
+      this.REQUIRES(Services.CREATOR_NODE)
+
       if (this.web3Manager.web3IsExternal()) {
-        this.REQUIRES(Services.CREATOR_NODE, Services.IDENTITY_SERVICE)
+        this.REQUIRES(Services.IDENTITY_SERVICE)
 
         phase = phases.CREATE_USER_RECORD
         await this.identityService.createUserRecord(email, this.web3Manager.getWalletAddress())
       } else {
-        this.REQUIRES(Services.CREATOR_NODE, Services.IDENTITY_SERVICE, Services.HEDGEHOG)
-
+        this.REQUIRES(Services.HEDGEHOG)
```

Comment:
pretty sure identity is required in both flows, since hedgehog writes to it. should uplevel tht require to line 120

Reply:
ah good call out. updating